### PR TITLE
chore: bump version of azuredisk to 0.5.1

### PR DESCRIPTION
**Pull request by hectorj2f**
**What type of PR is this?**

Chore
**What this PR does/ why we need it**:

Bump version to investigate volume disk deletion issues

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
blocked by https://github.com/mesosphere/charts/pulls
**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: addons/azuredisk-csi-driver/0.4.x/azuredisk-csi-driver-2.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: azuredisk-csi-driver
+  labels:
+    kubeaddons.mesosphere.io/name: azuredisk-csi-driver
+    kubeaddons.mesosphere.io/provides: csi-driver
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.5.0-1"
+    appversion.kubeaddons.mesosphere.io/azuredisk-csi-driver: "054.0"
```
Comment (alejandroEsc) [inline addons/azuredisk-csi-driver/0.4.x/azuredisk-csi-driver-2.yaml]: is this right? `054.0` or did you mean `0.54.0`

Comment (hectorj2f) [reply addons/azuredisk-csi-driver/0.4.x/azuredisk-csi-driver-2.yaml]: yep, you are right this is typo 👍

**Comment by shaneutt** [comment]
Need any help moving this to `ready` @hectorj2f ? just let us know if we can assist

**Comment by hectorj2f** [comment]
@shaneutt all ready to merge it

**Pull request merged**